addon-test-support/@ember/test-helpers/wait-until.js

```diff
@@ -0,0 +1,36 @@
+import { Promise } from 'rsvp';
+
+import { nextTick } from './-utils';
+
+export default function(callback, options = {}) {
+  let timeout = 'timeout' in options ? options.timeout : 1000;
+  let waitUntilTimedOut = new Error('waitUntil timed out');
```

Comment:
this could use a comment that the `Error` is created here to have the original stacktrace

Reply:
Sounds good. Will do that.